### ISSUE TITLE
Transfer over domain rights of `mybinder.org` to the Jupyter account

### ISSUE THREAD

**Issue by choldgraf** [jupyterhub/team-compass#163]
For the longest time we've spoken about transferring over the rights to `mybinder.org` because right now it's in a bus-factor-esque location on my personal Bluehost account. This issue is to try and keep track of this so we remember to make the transfer

## Who currently controls `mybinder.org`?

Chris does! That's a bad thing! Currently `mybinder.org` is registered under Bluehost with my personal account. We did this originally because it was the most practical way to quickly transfer the account from Jeremy to the Binder team. Now that Binder is stable and we are slowly improving our team practices, moving ownership of the domain is a good next step.

## Where we should be

I think that `mybinder.org` should be owned by the same account that owns `jupyter.org` and any other jupyter-related websites. I *think* that this is a Cloudflare account. Not sure if it's an individual account (maybe @Carreau?) or a group account. Either way, I think we should transfer ownership of mybinder.org there.

## What to do 

* Follow [the bluehost domain transfer instructions](https://my.bluehost.com/hosting/help/transferaway)
* Follow the receiving instructions for the receiving service
* Re-write all of the rules for transferring sub-domains
* Double-check that all still works OK.

## Thoughts or questions?

Anybody have thoughts or questions about this? I think we should pick a time when relatively little traffic is on mybinder.org, because I'm betting this will cause a disruption in service.

**Comment by minrk** [comment]
jupyter.org *DNS* is on a shared CloudFlare account, same as mybinder.org now, but I think the domain ownership itself is still on @fperez. Is that right?

**Comment by choldgraf** [comment]
Wait now I am confused...I was just able to create a cname on mybinder through the bluehost  interface so I assumed it was still controlling all the names and routing. Maybe I’m missing something?

**Comment by choldgraf** [comment]
OK, I just realized that I also have the cloudflare account details. And mybinder.org is there as well!

So I wonder if it's just that my bluehost account acts as if it can control the DNS, when it really cannot. Maybe that's what @minrk means about domain ownership vs. DNS rights?

**Comment by minrk** [comment]
Yeah, domain registration and DNS are separate-ish. It's in the registrar that you set the NS records, which tell the world to use cloudflare name servers for the domain. So it's still your bluehost acct that's relevant for renewal or moving from cloudflare to something else, but cloudflare for managing records themselves.

**Comment by betatim** [comment]
This means that access to the cloudflare account is what is needed to update the records for mybinder.org and there are already several people on the team who have access to that? In which case I think we can close this issue.

**Comment by choldgraf** [comment]
> In which case I think we can close this issue.

Though ownership of the domain is still under BlueHost, even though control is not. I'd still like to get ownership transferred over into an account w/ a higher bus factor than myself.

**Comment by yuvipanda** [comment]
My understanding is that right now, there's *no* one place that has ownership of the hosts. jupyter.org is still owned by Fernando.

So I *think* moving it to an account w/ a higher bus factor requires such an account to exist. I don't think one exists right now.

I could totally be wrong!

**Comment by Carreau** [comment]
The ownership of the domain should likely go to numfocus; the question is how much can we trust that their will be a plan to renew the domain when it expires. 

The domain should be "donated" to numfocus – in which case you can deduct the "price" of the domain from your taxes.

I'm tempted to think we should move the domain to something like google domain or whatever where you can have many owners/delegated rights.

**Comment by yuvipanda** [comment]
> I'm tempted to think we should move the domain to something like google domain or whatever where you can have many owners/delegated rights.

Big +1 to this! This also lets us separate question of bus factor from legal question of ownership.
